src/model/page.urls.ts

```diff
@@ -13,6 +13,10 @@ export const ENTER_INFORMATION: string = SEPARATOR + Templates.ENTER_INFORMATION
 export const DOCUMENT_UPLOAD: string = SEPARATOR + Templates.DOCUMENT_UPLOAD;
 export const DOCUMENT_UPLOAD_CONTINUE: string = SEPARATOR + "document-upload-continue"; // not a template
 export const REMOVE_DOCUMENT: string = SEPARATOR + Templates.REMOVE_DOCUMENT + "/:attachmentId";
+export const DOWNLOAD_ATTACHMENT: string =
+  "/company/:companyId/strike-off-objections/:requestId/attachments/*/download";
+export const DOWNLOAD_PREFIX: string = SEPARATOR + Templates.DOWNLOAD_ATTACHMENT_LANDING_PAGE;
```

Comment:
the prefix and the template name are 2 separate things even though they might share the same value. I would make this a string literal  SEPARATOR + "download"

Reply:
Replaced the template name reference with a string literal.